public/plugins/ckeditor/skins/bootstrapck/scss/components/_toolbar.scss

```diff
@@ -0,0 +1,317 @@
+    margin: -2px 4px 0 -2px;
+}
+
+/* This class is applied to the button when it is "active" (pushed).
+   This style indicates that the feature associated with the button is active
+   i.e. currently writing in bold or when spell checking is enabled. */
+.cke_button_on {
+    background: $primary-light;
+}
+
+.cke_hc {
+    .cke_button_on {
+        border-width: 3px;
+        /* Compensate the border change */
+        padding: 1px 3px;
+    }
+    a.cke_button_off, a.cke_button_disabled {
+        &:hover, &:focus, &:active {
+            border-width: 3px;
+            /* Compensate the border change */
+            padding: 1px 3px;
+        }
+    }
+}
+
+/* This class is applied to the button when the feature associated with the
+   button cannot be used (grayed-out).
+   i.e. paste button remains disabled when there is nothing in the clipboard to
+   be pasted. */
+.cke_button_disabled .cke_button_icon {
+    opacity: 0.3;
+}
+
+.cke_hc .cke_button_disabled {
+    opacity: 0.5;
+}
+
+a.cke_button_on {
+    &:hover, &:focus, &:active {
+    }
+}
+
+a.cke_button_off, a.cke_button_disabled {
+    &:hover, &:focus, &:active {
+        background: $primary-lighter;
+    }
+}
+
+/* The icon which is a visual representation of the button. */
+.cke_button_icon {
+    cursor: inherit;
+    background-repeat: no-repeat;
+    margin-top: 1px;
+    width: 16px;
+    height: 16px;
+    float: left;
+    display: inline-block;
+}
+
+.cke_rtl .cke_button_icon {
+    float: right;
+}
+
+.cke_hc .cke_button_icon {
+    display: none;
+}
+
+/* The label of the button that stores the name of the feature. By default,
+   labels are invisible. They can be revealed on demand though. */
+.cke_button_label {
+    display: none;
+    padding-left: 3px;
+    margin-top: 1px;
+    line-height: 18px;
+    vertical-align: middle;
+    float: left;
+    cursor: default;
+    color: $gray-dark;
+}
+
+.cke_rtl .cke_button_label {
+    padding-right: 3px;
+    padding-left: 0;
+    float: right;
+}
+
+.cke_hc .cke_button_label {
+    padding: 0;
+    display: inline-block;
+    font-size: 12px;
+}
+
+/* The small arrow available on buttons that can be expanded
+   (e.g. the color buttons). */
+.cke_button_arrow {
+    /* Arrow in CSS */
+    display: inline-block;
+    margin: 8px 0 0 1px;
+    width: 0;
+    height: 0;
+    cursor: default;
+    vertical-align: top;
+    border-left: 3px solid transparent;
+    border-right: 3px solid transparent;
+    border-top: 3px solid #474747;
+}
+
+.cke_rtl .cke_button_arrow {
+    margin-right: 5px;
+    margin-left: 0;
+}
+
+.cke_hc .cke_button_arrow {
+    font-size: 10px;
+    margin: 3px -2px 0 3px;
+    width: auto;
+    border: 0;
+}
+
+/* The vertical separator which is used within a single toolbar to split
+   buttons into sub-groups. */
+.cke_toolbar_separator {
+    float: left;
+    background-color: $hr-border;
+    margin: 4px 2px 0;
+    height: 16px;
+    width: 1px;
+}
+
+.cke_rtl .cke_toolbar_separator {
```

Comment:
Selector `cke_rtl` should be written in hyphenated BEM (Block Element Modifier) format<br>Selector `cke_toolbar_separator` should be written in hyphenated BEM (Block Element Modifier) format